CHANGELOG.adoc

```diff
@@ -40,6 +41,7 @@
 - https://github.com/eclipse-sirius/sirius-components/issues/985[#985] [core] Provide a variable to detect which environment is used. The value of the variable will change to something specific for the integrating application (for example siriusWeb). This is only available for the diagram for now
 - https://github.com/eclipse-sirius/sirius-components/issues/1025[#1025] [diagram] Add a new API to perform tests of our layout algorithm
 - https://github.com/eclipse-sirius/sirius-components/issues/699[#699] [core] Provide the `IEditingContext` to find all the `RepresentationMetadata` for a specific object
+- https://github.com/eclipse-sirius/sirius-components/issues/1065[#1065] [diagram] Stop adding start and end edges as routing points. Use default anchor positions or calculate them from ELK layout
```

Comment:
This should be reworded in terms of what concrete, visible improvement this has for users.

Reply:
I'll move that in the `Breaking changes` section since anchors are updated after an ELK layout